### PR TITLE
Untrack ipynb in linguist

### PR DESCRIPTION
**Pull request by janbridley**
## Motivation and Context
Github linguist autoselects the language for a repository. For coxeter, this results in the following:

image

Listing the package as primarily jupyter notebook could be confusing,and as far as I know there is no GLSL code. Updating the .gitattributes is a minor fix but a useful one nonetheless

## Types of Changes

- [ ] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/coxeter/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/coxeter/blob/master/ContributorAgreement.md).

### CODE REVIEW

**Comment by janbridley** [comment]
> It appears that https://github.com/glotzerlab/coxeter/blob/master/paper/figure1/shape_images/tri.txt.vert is the GLSL file:
> 
> https://github.com/github-linguist/linguist/blob/d190ad88af555bb2ee213aa64205f2b9a97a077d/lib/linguist/languages.yml#L2230-L2256

It should be possible to tell linguist to only run on the source directory - I'll try that

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+coxeter/* linguist-vendored
```
Comment (janbridley) [inline .gitattributes:1]:
```suggestion
paper/** linguist-vendored
```

**Comment by janbridley** [comment]
```bash
$ github-linguist
100.00% 360976     python
```

linguist-vendored untracks a folder in linguist, and subfolders must be added with parent/** instead of parent/*. This is all set now!

**Review by joaander — APPROVED** [review]
Thanks!

**Comment by janbridley** [comment]
Merge appears to be locked to admins only - I could override, but is there a systematic setting to change to fix this?

**Pull request merged**